README.markdown

```diff
@@ -34,9 +34,16 @@ Please contact Wenbin Yang, bysin7 (at) gmail (dot) com with questions or feedba
 
 Step by step install With Visual Studio 2017 for Windows 7/10 64 Bit
 -----------------------------------------------------------------
-* Install OpenCV 3.0 or later
-* Install OpenGL with GLFW3.0; (Check [this great website](https://learnopengl.com/) for more details.)
+* Install OpenCV 3.0 or later;Click [here](https://opencv.org/) and say hello to OpenCV
+  * Check this [link](https://docs.opencv.org/) and find what you need. There are many great tutorials in this website.
+  * If you configure all libs and includes with OpenCV successfully, installation of OpenGL and Pylon 5 will be very easy for you.
+    Other two processes are similar to this one.
+* Install OpenGL with GLFW3.0 or later
+  * Follow the tutorial in [this website](https://learnopengl.com/). Do not forget to install GLAD after installation of GLFW. It won't     cost you too much time.
+  * tips: You can download Windows pre-compiled binaries as an alternative. Click "64-bit Windows binaries" to download it. 
 * Install Balser Pylon 5
+  * Click [here](https://www.baslerweb.com/en/sales-support/downloads/software-downloads/) and download pylon Camera Software Suite.
+  * Check this [link](https://www.baslerweb.com/fp-1476182890/media/downloads/documents/application_notes/AW00136801000_Getting_Started_with_pylon5_and_OpenCV.pdf) to get started with         OpenCV and pylon.
```

Comment:
This one is a good tutorial to start with, however, it is not necessary. We have to make the instructions concise.

Reply:
Thanks, and I will edit it this afternoon.